pytorch_lightning/utilities/cli.py

```diff
@@ -48,6 +48,7 @@ def __init__(self, *args, parse_as_dict: bool = True, **kwargs) -> None:
         self.add_argument(
             '--config', action=ActionConfigFile, help='Path to a configuration file in json or yaml format.'
         )
+        self.callbacks = []
```

Comment:
More explicit name + typing

```suggestion
        self.callback_keys: List[str] = []
```